pandas/tests/frame/test_apply_relabeling.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
```

Comment:
can you make a sub-dir pandas/tests/frame/apply/test_relabling.py (and move the test_apply.py)

Reply:
can you do this

Reply:
done